test/test_auth.py

```diff
@@ -890,3 +892,136 @@ def test_peer_auth_ident_map(bouncer):
         host=f"{bouncer.admin_host}",
         user="bouncer",
     )
+
+
+async def test_auth_user_trust_auth_without_auth_file_set(bouncer) -> None:
+    """
+    This is a regression test for issue #1116, using the SET command
+    """
+    bouncer.admin("set auth_user='pswcheck_not_in_auth_file'")
+    bouncer.admin("set auth_type='trust'")
+    with bouncer.conn(
+        dbname="p7a",
+        user="pswcheck_not_in_auth_file",
+    ) as cn:
+        with cn.cursor() as cur:
+            cur.execute("select 1")
+
+
+def test_auth_user_trust_auth_without_auth_file_reload(bouncer) -> None:
+    """
+    This is a regression test for issue #1116, using the RELOAD command
+    """
+    config = f"""
+        [databases]
+        postgres = host={bouncer.pg.host} dbname=postgres port={bouncer.pg.port} min_pool_size=2
+
+        [pgbouncer]
+        listen_addr = {bouncer.host}
+        listen_port = {bouncer.port}
+        auth_type = trust
+        auth_user = pswcheck_not_in_auth_file
+        auth_dbname = postgres
+        admin_users = pgbouncer
+        logfile = {bouncer.log_path}
+        auth_file = {bouncer.auth_path}
+    """
+
+    with bouncer.run_with_config(config):
+        with bouncer.conn(
+            dbname="postgres",
+            user="postgres",
+        ) as cn:
+            with cn.cursor() as cur:
+                cur.execute("select 1")
+
+
+def test_auth_user_at_db_level_trust_auth_without_auth_file_reload(bouncer) -> None:
+    """
+    This is a regression test for issue #1116, when auth_user was set at the
+    database level
+    """
+    config = f"""
+        [databases]
+        postgres = host={bouncer.pg.host} dbname=postgres port={bouncer.pg.port} min_pool_size=2 auth_user=pswcheck_not_in_auth_file
+
+        [pgbouncer]
+        listen_addr = {bouncer.host}
+        listen_port = {bouncer.port}
+        auth_type = trust
+        auth_dbname = postgres
+        admin_users = pgbouncer
+        logfile = {bouncer.log_path}
+        auth_file = {bouncer.auth_path}
+    """
+
+    with bouncer.run_with_config(config):
+        with bouncer.conn(
+            dbname="postgres",
+            user="pswcheck_not_in_auth_file",
+        ) as cn:
+            with cn.cursor() as cur:
+                cur.execute("select 1")
+
+
+def test_auth_user_with_same_forced_user(bouncer):
+    """
+    Check that the pgbouncer correctly handles multiple credentials with the
+    same name with a global auth_user (isue #1103).
```

Comment:
Hi!

Could you correct "isue" with "issue", too?

I am sorry for this mistype.

Thanks.